src/auth/auth.service.ts

```diff
@@ -2,10 +2,19 @@ import { Injectable } from '@nestjs/common';
 import { UsersService } from '../users/users.service';
 import { JwtService } from '@nestjs/jwt';
 
+var randtoken = require('rand-token');
```

Comment:
We should use the jwt package rather than a rand-token package. Also import statements are better than require statements - keeps the overall application more lightweight